tracks/bash/exercises/hamming/mentoring.md

```diff
@@ -2,22 +2,23 @@
 ```bash
 #!/usr/bin/env bash
 main () {
-  (( "$#" == 2 )) || {
+  local -i count
+
+  if (( "$#" != 2 )); then
```

Comment:
Should we make a mentor note regarding this kind of change?  It makes sense to me since the reason is in the message of this pull request.

Reply:
What sort of note? The downsides of using `&&`/`||` in place of an `if`? It's listed in the notes already.

Reply:
> Drop advise that leans towards bad practice (using cmd && { .. } over if cmd; then .. fi).

Something that advises the reason to go away from this in mentor notes, if it is part of the student solution.

Reply:
>  It's listed in the notes already.

I checked, but I must have missed it.

Reply:
Talking Points => List Constructs used instead of if/else